deployment/keystone/deploy.go

```diff
@@ -137,6 +145,85 @@ func DeployContracts(lggr logger.Logger, e *deployment.Environment, chainSel uin
 	}, nil
 }
 
+// DonInfo is DonCapabilities, but expanded to contain node information
+type DonInfo struct {
+	Name         string
+	Nodes        []Node
+	Capabilities []kcr.CapabilitiesRegistryCapability // every capability is hosted on each node
+}
+
+type Node struct {
```

Comment:
this looks product agnostic - should be able to combine/merge with https://github.com/smartcontractkit/chainlink/blob/c6c8fbd559091bd3fb6673c7fb20c90a92aebe1d/deployment/environment.go#L217?